manifest.yml

```diff
@@ -8,4 +8,5 @@ applications:
   name: language-translation-nodejs
   command: node app.js
   path: .
-  memory: 256M
+  memory: 512M
+  instances: 2
```

Comment:
That's what German added to the running demo, I also wasn't sure why, but I presume he saw some issues and decided to play safe.